ReactWindows/ReactNative/UIManager/AppRegistry.cs

```diff
@@ -13,7 +13,7 @@ public sealed class AppRegistry : JavaScriptModuleBase
         /// </summary>
         /// <param name="appKey">The app key.</param>
         /// <param name="appParameters">The app parameters.</param>
-        public void runApplication(string appKey, JObject appParameters)
+        public void runApplication(string appKey, IDictionary<string, object> appParameters)
```

Comment:
Add a comment to keep the casing on this and other JavaScript modules.  I believe the method calls into JavaScript are case sensitive.